packages/desktop/src/providers/variables.rs

```diff
@@ -20,4 +20,6 @@ pub enum ProviderVariables {
   Memory(MemoryVariables),
   Network(NetworkVariables),
   Weather(WeatherVariables),
+  #[cfg(all(windows, target_arch = "x86_64"))]
```

Comment:
should work fine on arm64 (and we don't compile for 32bit), so can be simplified to just `#[cfg(windows)]`